.gitignore

```diff
@@ -5,3 +5,6 @@
 
 # OS files
 .DS_Store
+
+#Ignore saved plot files
+*.png
```

Comment:
Best practice is to end the file with a blank line please.

Reply:
Also (I'm picky about these kind of things) please add a space after `#`

Reply:
These have been changed and committed to the branch.